Makefile

```diff
@@ -17,14 +17,16 @@ SO = so
 INSTALL += -D
 endif
 
-all: libsfark.$(SO)
+all: libsfark.$(SO).0.0.0
```

Comment:
Let's put the major/minor/patchlevel version parts in variables so we don't have to repeat them.

In general I'm not sure version numbers belong in source control, I'd prefer them to be (only) in tags, but I suppose this is an improvement  :)

Reply:
Yes, these are badly done. I’ll try to update this.